cumulusci/core/github.py

```diff
@@ -51,6 +51,7 @@ def get_github_api_for_repo(keychain, owner, repo):
                 raise GithubException(
                     f"Could not access {owner}/{repo} using GitHub app. "
                     "Does the app need to be installed for this repository?"
+                    "\nUse the `cci service list` command to show what services are currently configured."
```

Comment:
@Br4nd0R in this code path (starting line 44), CumulusCI is using a GitHub app configured using the GITHUB_APP_ID and GITHUB_APP_KEY environment variables, not an individual user's username and password configured in the github service.

Reply:
Ahh, ya. Not applicable here. Will close this out.